From 8d0f679aaef21460244a20e063ccf08e08876f2a Mon Sep 17 00:00:00 2001
From: Saalim Quadri <danascape@gmail.com>
Date: Fri, 25 Sep 2026 00:41:53 +0530
Subject: [PATCH] SUW: Supply a default for the card view intrinsic dimensions

  Resources$NotFoundException: Resource ID #0x7f0704b8
    at PartnerConfigHelper.getTypedValueFromResource
    at PartnerConfigHelper.getDimension
    at IntrinsicSizeFrameLayout.init

Assisted-By: LLM
Change-Id: I87808806a534bad78135679eaee1e78cadb27863
Signed-off-by: Saalim Quadri <danascape@gmail.com>
---

diff --git a/app/src/main/res/values/dimens.xml b/app/src/main/res/values/dimens.xml
index acd9ce8..92b1be9 100644
--- a/app/src/main/res/values/dimens.xml
+++ b/app/src/main/res/values/dimens.xml
@@ -36,6 +36,8 @@
 
     <dimen name="setup_compat_footer_button_radius">@dimen/sud_glif_expressive_footer_button_radius</dimen>
     <dimen name="setup_design_items_group_corner_radius">20dp</dimen>
+    <dimen name="setup_design_card_view_intrinsic_width">0dp</dimen>
+    <dimen name="setup_design_card_view_intrinsic_height">0dp</dimen>
 
     <item name="finish_logo_scale" format="float" type="dimen">1.2</item>
     <dimen name="swipe_hint_icon_size">67dp</dimen>
